packages/devops/scripts/ci/triggerRedeploy.sh

```diff
@@ -21,8 +21,9 @@ fi
 
 echo "At least one running deployment, triggering redeploy of any tagged with Branch ${CI_BRANCH}"
 RESPONSE_FILE=lambda_redeploy_response.json
+# TODO: Set the 'testRDS' to a correct name before merging rn-195-epic
```

Comment:
So the current plan is to keep this as is. Otherwise we'd need to replace the existing deployment lambda with the new one, which will put us in a tricky spot if anything goes wrong with the merge (which it may very well...). So I was thinking we keep both lambda functions around, and then when we've confirmed that the new function is working correctly we switch this to `deployment`?